packages/garbo/src/config.ts

```diff
@@ -83,6 +85,10 @@ const defaultTarget =
 export const targettingMeat = () =>
   $monsters`cheerless mime executive, cockroach`.includes(globalOptions.target);
 
+export const targettingItems = () =>
+  have($familiar`Grey Goose`) &&
+  $monster`Witchess Knight` === globalOptions.target;
```

Comment:
it'd be more robust if we checked the monsters drops for items that can be duped

Reply:
Is there an easy way to do this? It doesn't seem to be any of the proxy records assigned to items.

Reply:
[itemDropsArray()](https://wiki.kolmafia.us/index.php/Item_drops_array) something similar to how we find [good free fights from the locket](https://github.com/loathers/garbage-collector/blob/815600b94a32e71c34b944c9597dee31e888586c/packages/garbo/src/tasks/freeFight.ts#L98) i'd wager

Reply:
You'd ideally want a function that maps the itemDropsArray() for each garboValue item and determine the profitability of emitting drones - but then (as Scotch noted) you'd also need to plan for more than one drone AND ensure that a constantValueFamiliar wasn't just better.

All in all, it would take a lot to be done well.